easybuild/easyconfigs/j/Java/Java-1.8.0_141.eb

```diff
@@ -0,0 +1,20 @@
+name = 'Java'
+version = '1.8.0_141'
```

Comment:
@JackPerdue worth updating to `1.8.0_144`, or did you need this specific version?